Related to issue #48 

If we don't fix this, we will encounter header file missing error like this:
https://ci.appveyor.com/project/holishing/pcman-windows/builds/33028221#L4854

```
c:\projects\pcman-windows\lite\termview.cpp(1994): warning C4996: 'gethostbyname': Use getaddrinfo() or GetAddrInfoW() instead or define _WINSOCK_DEPRECATED_NO_WARNINGS to disable deprecated API warningsc:\tools\vcpkg\installed\x86-windows\include\cpprest\ws_client.h(36): fatal error C1083: Cannot open include file: 'boost/asio/ssl.hpp': No such file or directory 
```

--- a/Dependencies.txt
+++ b/Dependencies.txt
@@ -1,17 +1,16 @@
 ﻿這份文件說明開發和釋出的需求
 
-- [開發] Visual Studio 2017 (Community 即可)
+- [開發] Visual Studio 2017/2019 (Community 即可)
 
 請至官方網站下載: https://www.visualstudio.com/
 
-- [開發] cpprestsdk
+- [開發] cpprestsdk (須包含 websockets 這個 feature)
 
-這兩個推薦使用 vcpkg 安裝, 安裝說明請至以下網站:
+推薦使用 vcpkg 安裝, 安裝說明請至以下網站:
 https://github.com/Microsoft/vcpkg
 
-安裝指令: vcpkg install cpprestsdk
+安裝指令: vcpkg install cpprestsdk[default-features,websockets]
 
-- [釋出] Microsoft Visual C++ Redistributable for Visual Studio 2017 (或相對應的版本)
+- [釋出] Microsoft Visual C++ Redistributable for Visual Studio 2017/2019 (或相對應的版本)
 
-可至 https://www.visualstudio.com/downloads/ 的
-Other Tools and Frameworks 區域下載.
+可至 https://www.visualstudio.com/downloads/ 的 Other Tools and Frameworks 區域下載.

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,5 +1,7 @@
 version: '{build}'
-image: Visual Studio 2017
+image: 
+  - Visual Studio 2017
+  - Visual Studio 2019
 configuration: Release
 init:
 - ps: Set-Culture zh-TW
@@ -10,7 +12,7 @@ install:
 - .\bootstrap-vcpkg.bat
 - cd "%APPVEYOR_BUILD_FOLDER%"
 - vcpkg remove --outdated --recurse
-- vcpkg install cpprestsdk
+- vcpkg install cpprestsdk[default-features,websockets]
 
 - ps: .\Scripts\BeforeBuild.ps1
 clone_folder: C:\projects\pcman-windows

